libraries/shared/storage/decoder.go

```diff
@@ -108,6 +118,6 @@ func getNumberOfBytes(valueType types.ValueType) int {
 	case types.Address:
 		return 20
 	default:
-		panic(fmt.Sprintf("ValueType %d not recognized", valueType))
+		return 0
```

Comment:
Maybe worth returning `(int, error)` from this function instead of making callers handle `0` as a special case?